packages/utils/src/safe_math_revert_errors.ts

```diff
@@ -10,7 +10,7 @@ export enum SafeMathErrorCodes {
 }
 
 export class SafeMathError extends RevertError {
-    constructor(error?: SafeMathErrorCodes, a?: BigNumber | number | string, b?: BigNumber | number | string) {
+    constructor(error?: SafeMathErrorCodes, a?: BigNumber, b?: BigNumber) {
```

Comment:
`string`s are a little sketchy, but what was wrong allowing `number`s?

Reply:
I just did this for consistency. I've noticed that in a lot of places we just accept BigNumbers, which I kind of like. If you feel strongly about this, I can change it back